pkg/controller/quayecosystem/provisioning/provision.go

```diff
@@ -441,6 +439,8 @@ func (r *ReconcileQuayEcosystemConfiguration) createQuayRoute(meta metav1.Object
 		return err
 	}
 
+	time.Sleep(time.Duration(2) * time.Second)
+
```

Comment:
This seems sketchy... Is there no way to check for existence? Perhaps a loop-catch-error-N-times instead?

Reply:
if there is an error, it will just trigger another reconcile loop (similar to a try/catch loop). This is here just to wait for the API server to accept the route (should cover 90% of the cases)